python/README.md

```diff
@@ -53,3 +53,35 @@ properties = {
 
 chemiscope.show(frames=frames, properties=properties)
 ```
+
+To create a new chemiscope input file using `stk`_:
+
+.. _`stk`: https://github.com/lukasturcani/stk
+
+`stk` is an optional dependancy:
+
+```bash
+pip install stk
+```
+
+```python
+import chemiscope
+import stk
+
+# read frames using ase
+frames = [stk.BuildingBlock(smiles="NCCN")]
+
+# add additional properties to display
+properties = {
+    "<property name>": {
+        target: "atom",
+        values: [3, 4, 2, 8, 9, 10],
+    }
+}
+
+chemiscope.write_input(
+    "my-input.json.gz",
+    frames=frames,
+    properties=properties,
+)
+```
```

Comment:
Instead of a whole new example here, I would add a comment above that the structures/frames can also come from STK in addition to ASE, something like

```
# frames can also be STK objets, e.g.
# frames = [stk.BuildingBlock(smiles="NCCN")]

```

Reply:
Done.